install/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Installing</title>
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Installing OpenShift Service Mesh</subtitle>
+<abstract>
+    <para>This documentation provides information about installing OpenShift Service Mesh 3.0.
+    </para>
+</abstract>
+<authorgroup>
+    <orgname>Red Hat OpenShift Documentation Team</orgname>
```

Comment:
If "3.0" is the {product-version}, you could use the attribute.

Reply:
I created an attribute and used it in a context where I think it makes sense to do so.